backend/src/main/scala/cromwell/backend/model/MemorySize.scala

```diff
@@ -1,4 +1,4 @@
-package cromwell.backend.validation
+package cromwell.backend.model
```

Comment:
We are not doing MVC. What I'm trying to say is that is common to put all VOs that represents the model for that component together as I shown in the examples and it's common as well to create a folder called "model" even if you don't implemented a MVC or MVCS or MVP pattern.